mpc-recovery/src/error.rs

```diff
@@ -71,6 +71,8 @@ pub enum LeaderNodeError {
     RelayerError(#[from] RelayerError),
     #[error("recovery key can not be deleted: {0}")]
     RecoveryKeyCanNotBeDeleted(PublicKey),
+    #[error("action can not be performed, account deletion is not allowed")]
+    AccountDeletionIsNotAllowed,
```

Comment:
nit: `IsNotAllowed` seems a little funky to type out. How about `Unsupported` since it's not as rough to type out with fewer capitalizations and one letter shorter